ranked-choice-voting/index.py

```diff
@@ -32,7 +32,7 @@ def create_dataset_for_modeling(num_ballots_to_generate):
     # Vectorizing dependent var:
     master_df['spoiled'] = np.where(master_df['spoiled'] == 'Y', 0, 1)  # 0 = yes, 1 = no
 
-    master_df.to_csv('final_master.csv', index=False)
+    master_df.to_csv('final_master_23.csv', index=False)
```

Comment:
I wouldn't commit this type of thing, since it only applies to your local implementation. Can we change back to just `final_master`?